config/mcu_definitions/toolhead/BTT_SB2240_v1.0.cfg

```diff
@@ -0,0 +1,21 @@
+[board_pins toolhead_manufacturer]
+mcu: toolhead
+aliases:
+   MCU_MOTOR_SPI_NSS=PA15 , # CLK/MOSI/MISO are shared with SPI2 configuration
```

Comment:
On this board, there is also UART available so we can add it to the manufacturer file I think: `MCU_MOTORDRIVE_STEP=PD0`, `MCU_MOTORDRIVE_DIR=PD1`, `MCU_MOTORDRIVE_EN=PD2` and `MCU_MOTORDRIVE_DIAG=PB3`

Reply:
and where is the UART pin? 
BTT only defines the tmc2240 for SPI in their config. So i'd rather not include something that is not meant to be used.

Reply:
Yeah ok no problem